README.rst

```diff
@@ -134,6 +134,23 @@ If you don't have tox, you can generate a pex of tox:
 
     $ pex tox -c tox -o ~/bin/tox
 
+To run a specific test command, look in `tox.ini` for the name and run like this:
```

Comment:
There is also `tox -a`.

Reply:
My search skills are failing me..can't find what this means either on Tox's site or duckduckgo/google search. Mind explaining?

Tox's docs seem to use `-e` everywhere I find.

Reply:
Try running `tox --help`.

Reply:
Ah yes that would do the trick... Thanks!